### PR TITLE
Remove deprecated services

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Remove deprecated services
| Type?             | improvement
| Category?         | CO
| BC breaks?        | yes
| Deprecations?     | no
| How to test?      | CI is :green_circle: & Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/5653961045
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

## BC Breaks 
* Removed service `prestashop.adapter.admin.page_preference`
* Removed service `prestashop.adapter.attachment.attachment_repository`
* Removed service `prestashop.adapter.product.attribute_group.attribute_group_view_data_provider`
* Removed service `prestashop.adapter.backup.database_creator`
* Removed service `prestashop.adapter.backup.backup_provider`
* Removed service `prestashop.adapter.backup.backup_remover`
* Removed service `prestashop.adapter.category.repository.category_repository`
* Removed service `prestashop.adapter.currency.repository.currency_repository`
* Removed service `prestashop.adapter.data_provider.tab`
* Removed service `prestashop.adapter.image.image_validator`
* Removed service `prestashop.adapter.admin.data_provider.product`
* Removed service `prestashop.adapter.admin.data_updater.product`
* Removed service `prestashop.adapter.admin.wrapper.product`
* Removed service `prestashop.adapter.admin.controller.attribute_generator`
* Removed service `prestashop.adapter.product.filter_categories_request_purifier`
* Removed service `prestashop.adapter.product.list_parameters_updater`
* Removed service `prestashop.adapter.image_manager`
* Removed service `prestashop.adapter.tax.tax_computer`
* Removed service `prestashop.adapter.tax_rules_group.repository.tax_rules_group_repository`
* Removed service `form.form_cloner`
* Removed service `prestashop.core.admin.feature_flag.repository`
* Removed service `prestashop.bundle.routing.converter.router_provider`
* Removed service `prestashop.utils.float_parser`
* Removed service `prestashop.core.category.name_builder.category_display_name_builder`
* Removed service `prestashop.core.domain.country.zip_code_format_resolver`
* Removed service `prestashop.core.grid.position.position_update_factory`
* Removed service `prestashop.core.grid.position.doctrine_grid_position_updater`
* Removed service `prestashop.core.help.documentation`
* Removed service `prestashop.core.admin.module.repository`
* Removed service `prestashop.module.sourcehandler.zip`
* Removed service `prestashop.module.manager`
* Removed service `prestashop.core.product.csv_exporter`
* Removed service `prestashop.core.util.internationalized_domain_name_converter`
* Removed service `prestashop.core.uti.back_url_provider`
* Removed service `prestashop.core.util.arabic_to_latin_digit_converter`
* Removed service `prestashop.core.util.url.url_file_checker`
* Removed service `prestashop.core.util.string.string_validator`
* Removed service `prestashop.core.util.string.string_modifier`
* Removed service `prestashop.core.util.file_size.size_converter`
* Removed service `prestashop.core.util.helper_card.documentation_link_provider`

### CODE REVIEW

File: src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml [hunk 1]
```diff
@@ -775,37 +775,3 @@ services:
   PrestaShopBundle\Form\Admin\Type\TaxInclusionChoiceType:
   PrestaShopBundle\Form\Admin\Type\ShippingInclusionChoiceType:
   PrestaShopBundle\Form\Admin\Type\ProductSearchType:
-
```
Comment (FabienPapet) [inline src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml]: Attention, you're removing service that has been deprecated **in** 9.0

**Review by FabienPapet — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml (hunk 1 above)
Comment (Progi1984) [reply src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml]: Sorry. That lacks some comments.

**Review by boherm — DISMISSED** [review]
LGTM, thanks @Progi1984!

**Comment by Progi1984** [comment]
@boherm I just rebased

**Comment by Progi1984** [comment]
Hi @PrestaShop/committers, I had one approval for @boherm (but it dismiss after rebase). Can I have two reviews when you will have some availability, plz ? Thanks

**Review by boherm — DISMISSED** [review]

**Review by matks — APPROVED** [review]

**Review by kpodemski — DISMISSED** [review]
Conflicts

**Review by matks — DISMISSED** [review]
Apologies @Progi1984 I just reviewed the PR and the content seems OK for me however it now needs to be rebased

**Comment by Progi1984** [comment]
@PrestaShop/committers / @PrestaShop/prestashop-maintainers Rebased, a new time :)

**Review by boherm — DISMISSED** [review]
LGTM

**Review by matks — DISMISSED** [review]

**Review by kpodemski — COMMENTED** [review]
Conflict again. 

Let's do it this way, after the next rebase, please ping us on Slack so we can review it ASAP.

**Review by kpodemski — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by Progi1984** [comment]
@PrestaShop/qa-functional Automated tests are OK.

**Comment by Progi1984** [comment]
Thanks @Robin-Fischer-PS :heart_eyes_cat:

**Pull request merged**